src/main/io/vtx_smartaudio.c

```diff
@@ -279,9 +279,7 @@ static void saProcessResponse(uint8_t *buf, int len)
         // saDevice.version = 0 means unknown, 1 means Smart audio V1, 2 means Smart audio V2 and 3 means Smart audio V2.1
         saDevice.version = (buf[0] == SA_CMD_GET_SETTINGS) ? 1 : ((buf[0] == SA_CMD_GET_SETTINGS_V2) ? 2 : 3);
         saDevice.channel = buf[2];
-        if(saDevice.version != 3) {
-            saDevice.power = buf[3];
-        }
+        uint8_t rawpowervalue = buf[3];
```

Comment:
Can you please use camelCase for your variable names?

Reply:
Coding convention always gets me. Thanks for your patience, ill change it.